soepy/exogenous_processes/children.py

```diff
@@ -32,7 +32,11 @@ def gen_prob_child_vector(model_spec):
 
     # Read data frame with information on probability to get a child
     # in every period
-    exog_child_info_df = pd.read_pickle(model_spec.kids_info_file_name)
+    exog_child_info_df = pd.read_pickle(model_spec.child_info_file_name)
+
+    exog_child_info_df = exog_child_info_df.iloc[
```

Comment:
Please us proper slicing for this step here and in the other cases.  See the documentation starting here https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#selection-by-label